docs/troubleshooting.md

```diff
@@ -0,0 +1,59 @@
+## Whitelhall troubleshooting
+
+You may run into errors while working locally and trying to render pages on the Frontend.
+
+Problem
+
+```
+Mysql2::Error: Access denied for user 'whitehall'@'localhost' (using password: YES)
+```
+
+Solution:
+Login, create user, give access & quit
+
+```
+mysql -u root -p
```

Comment:
I'm assuming you'd run these on the docker image? Could we at least reference instructions for how to get into the image at the top?

Reply:
Good shout

Reply:
So... I'm expecting there to be some sort of `exec`ing into the docker image or maybe a `govuk-docker-run` or something as part of these commands. Surely running `mysql` locally is something that we don't want to encourage since it's all in the docker container?